contracts/extensions/GovernorSequentialProposalIdUpgradeable.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {GovernorUpgradeable} from "contracts/extensions/GovernorUpgradeable.sol";
+
+abstract contract GovernorSequentialProposalIdUpgradeable is GovernorUpgradeable {
```

Comment:
document purpose of this contract (like, how it's mostly a copy of GovernorStorageUpgradeable) -- creating issue